releasenotes/notes/0.16/timeline-drawer-1c5e9b4a7f50f8c8.yaml

```diff
@@ -15,4 +15,4 @@ features:
       qc = QuantumCircuit(2)
       qc.h(0)
       qc.cx(0,1)
-      timeline_drawer(transpile(qc, FakeAthens(), scheduling_method='alap'))
+      timeline_drawer(transpile(qc, FakeAthens(), scheduling_method='alap', layout_method="trivial"))
```

Comment:
The fact that we need to change something in an old release note seems like something that's going to bite us horribly in the future with removals.  Perhaps we should look to replace `.. jupyter-execute` with non-execution blocks in here?  It's not ideal, though.

Reply:
Sure we can change this to a code-block. I did this because of the `FutureWarning` it's not actually a change in behavior yet, but the `FutureWarning` in the docs build was fatal because we run sphinx-build with `-W`.

Reply:
I was less concerned about this specific use, and more about _all_ uses of `.. jupyter-execute` in the release notes - it's bitten us once here, but it's likely that there's other behaviour in the release notes that will be changed or removed in the future (I count ~80 uses of `jupyter-execute`).  Perhaps we should think about _only_ using `code-block` in the release notes.